ui/src/app/applications/components/application-node-info/application-node-info.tsx

```diff
@@ -120,7 +120,9 @@ export const ApplicationNodeInfo = (props: {
                                 <YamlEditor
                                     input={live}
                                     hideModeButtons={!live}
-                                    onSave={(patch, patchType) => services.applications.patchResource(props.application.metadata.name, props.node, patch, patchType)}
+                                    onSave={(patch, patchType) =>
+                                        services.applications.patchResource(props.application.metadata.name, props.application.metadata.name, props.node, patch, patchType)
```

Comment:
Should second argument to this fxn call be `props.application.metadata.namespace`?

Reply:
Good catch! Fixed.